backend/src/main/java/com/traders/backend/controller/AdminOperationController.java

```diff
@@ -0,0 +1,24 @@
+package com.traders.backend.controller;
+
+import com.traders.backend.model.User;
+import com.traders.backend.repository.UsersRepository;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/admin")
+public class AdminOperationController {
+    private final UsersRepository usersRepository;
+
+    public AdminOperationController(UsersRepository usersRepository) {
+        this.usersRepository = usersRepository;
+    }
+
+    @RequestMapping(value = "/allusers", method = RequestMethod.GET)

```

Comment:
I think the path should be just "users" instead of "allusers", since we are going for a RESTful API.